exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -159,6 +163,14 @@ public void start() throws Exception {
       servletContextHandler.setSessionHandler(createSessionHandler(servletContextHandler.getSecurityHandler()));
     }
 
+    if (config.getBoolean(ExecConstants.HTTP_ENABLE_CORS)) {
+      FilterHolder cors = servletContextHandler.addFilter(CrossOriginFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
+      cors.setInitParameter(CrossOriginFilter.ALLOWED_ORIGINS_PARAM, "*");
```

Comment:
+1 that may pose a security risk. We should extend the configuration with allowed-origins. Use of * is too relaxed.